packages/core/src/payment/payment.ts

```diff
@@ -23,6 +24,7 @@ export type PaymentInstrument =
     | FormattedPayload<
           | AdyenV2Instrument
           | AppleInstrument
+          | BlueSnapDirectEcpPayload
```

Comment:
Is there a reason we need to declare this here? And not in the integration package itself?

Reply:
Because this line won't recognize it as a valid payload:

https://github.com/bigcommerce/checkout-sdk-js/blob/3edf18d171602a064f939c3ba90c74b609881113/packages/core/src/payment-integration/default-payment-integration-service.ts#L106

```
Argument of type '/packages/payment-integration-api/src/order/order-request-body'
    is not assignable to parameter of type
        '/packages/core/src/order/order-request-body'
```

`submitOrder`'s `payload` parameter still points to core's `OrderRequestBody`

Reply:
Can you add this in the package itself, something like `withBlueSnapDirectEcpPayload` and get the build system to import it. Or put this in payment-integration-api package.

Reply:
For example https://github.com/bigcommerce/checkout-sdk-js/blob/master/packages/core/src/order/order-request-body.ts#L1

Reply:
Done!